gnosis/eth/multicall.py

```diff
@@ -65,14 +78,17 @@ def __init__(
         ethereum_network = ethereum_client.get_network()
         address = multicall_contract_address or self.ADDRESSES.get(ethereum_network)
         if not address:
-            raise EthereumNetworkNotSupported(
-                "Multicall contract not available for %s", ethereum_network.name
-            )
+            # Try with Multicall V3 deterministic address
+            address = self.ADDRESSES.get(EthereumNetwork.MAINNET)
```

Comment:
always mainnet?

Reply:
It's a deterministic address and they tried to use MAINNET address everywhere